### PR TITLE
Redo of previous pull request. Database and model update

### PR DESCRIPTION
**Pull request by jesseDtucker**
Same as previous pull request. Please see #63 for details.

### CODE REVIEW

**Comment by jesseDtucker** [comment]
Fixing more conflicts....

**Comment by jesseDtucker** [comment]
Right, ready to merge. Let me know what you think.

**Comment by jesseDtucker** [comment]
Hold off.... fixing some other issues... Some views were bypassing the controller....

**Comment by jesseDtucker** [comment]
Issues fixed. I want to take another look at the view->controller interaction as there was some business logic there but not important for today. Also tested

File: adventure.datetime/src/ca/cmput301f13t03/adventure_datetime/controller/UserController.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2013 Andrew Fontaine, James Finlay, Jesse Tucker, Jacob Viau, and
+] * Copyright (c) 2013 Andrew Fontaine, James Finlay, Jesse Tucker, Jacob Viau, and
```
Comment (afontaine) [inline adventure.datetime/src/ca/cmput301f13t03/adventure_datetime/controller/UserController.java:2]: Silly open bracket

**Comment by afontaine** [comment]
Looks good to me, tested and works. :shipit:

**Comment by edegraff** [comment]
Tested and approved, looks good to merge.

**Pull request merged**